applications/HDF5Application/README.md

```diff
@@ -28,7 +28,7 @@ The HDF5 C libraries are used with the *HDF5Application*. If Kratos is configure
 2. Configure Kratos to build the *HDF5Application*
 
     ```
-    -DHDF5_APPLICATION=ON \
+    add_app ${KRATOS_APP_DIR}/HDF5Application
```

Comment:
this won't work for win, probably better to point to the corresponding section of INSTALL.md
https://github.com/KratosMultiphysics/Kratos/blob/master/INSTALL.md#adding-applications

Reply:
True

Reply:
Done